### PR TITLE
MVP

### PR DESCRIPTION
**Pull request by ianpaulfo**
imported Router, wrapped App, imported Link, updated syntax, created routes for each component: MVP

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import React from "react";
+import "./App.css";
+import { Home, About, Contact, Navigation } from "./components";
+import { Route } from "react-router-dom";
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
+    <Route exact path="/about" component={About} />
```
Comment (JossWritesCode) [inline src/App.js:10]: by the way, using "exact" here isn't necessary but also doesn't hurt anything

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
-import { Home, About, Contact, Navigation } from './components';
+import React from "react";
+import "./App.css";
+import { Home, About, Contact, Navigation } from "./components";
+import { Route } from "react-router-dom";
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
```
Comment (JossWritesCode) [inline src/App.js:9]: Looks great. Fun fact: did you know that "exact" is the same as exact={true} in this case? You could also explicitly write "exact={false}" or just leave exact out.

File: src/components/Navigation.js [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+import { Link } from "react-router-dom";
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/">Home</Link>
```
Comment (JossWritesCode) [inline src/components/Navigation.js:10]: Pop Quiz: What's the difference between Link and NavLink?

**Review by JossWritesCode — COMMENTED** [review]
Fantastic! Nicely done!

File: src/components/Navigation.js (hunk 3 above)
Comment (ianpaulfo) [reply src/components/Navigation.js:10]: NavLink includes props to style the selected route

**Pull request merged**